### PR TITLE
travis: parallel testing

### PR DESCRIPTION
**Pull request by kaplun**
Signed-off-by: Samuele Kaplun <samuele.kaplun@cern.ch>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -31,6 +31,11 @@ language: python
 python:
   - 2.7
 
+env:
+  - PYTEST="tests/test_views.py"
+  - PYTEST="tests/test_workflows.py"
+  - PYTEST="--ignore tests/test_views.py --ignore tests/test_workflows.py"
```
Comment (jacquerie) [inline .travis.yml]: I don't like the harcoded test filenames. A better solution (IMO) is to separate the tests in `integration` and `unit` folders, and run those separately.

Comment (kaplun) [inline .travis.yml]: Ouch :dancer: I have just seen it after I clicked merge...

Comment (jacquerie) [inline .travis.yml]: No worries! I will fix this eventually, as long as `Bibtex` doesn't kill me first...

Comment (kaplun) [inline .travis.yml]: BTW: sure. This was just a quick thing to gain 5 minutes in all the next testing. On the other hand, Travis testing is as slow as the slowest of its matrix component, so if you break up unit testing and integration testing only, the bottle neck will still be integration. We would still need to break it up a bit.

Comment (jacquerie) [inline .travis.yml]: Sure! Just for laughs, the problem of splitting them in the optimal way is NP-complete: https://en.wikipedia.org/wiki/Partition_problem, so maybe we shouldn't sweat it...

**Pull request merged**